RELEASENOTES.docu

```diff
@@ -1706,4 +1706,7 @@ extern typedef struct { } my_type_t;</pre> </add-note></build-id>
   <build-id value="6143"><add-note> Implicitly initialized <tt>local</tt>
       variables of (partially) const-qualified struct type no longer result in
       invalid generated C <bug number="SIMICS-10197"/>.</add-note></build-id>
+  <build-id value="next"><add-note> Any error encountered when evaluating the
+      operand to <tt>sizeof</tt> will now cause DMLC to suggest
+      <tt>sizeoftype</tt> <bug number="SIMICS-17991"/>.</add-note></build-id>
```

Comment:
Is this releasenote warranted? On one hand, it has no bearing outside of error messages; on the other hand, it does close an issue.

Reply:
It's warranted if it closes an *externally reported* issue.

Reply:
It's not; internally reported by Jakob. I'm removing this note, then.